### ISSUE TITLE
"Contact this Recruiter" functionality from JS dashboard is not correct

### ISSUE THREAD

**Issue by achille005** [achille005/JobApp#27]
Since "Contact this Job Seeker" is a paid feature letting the Job Seeker contact  the recruiter via email does not profit us! 2 solutions to this problem:
- send this job seeker details and resume profile to the Recruiter dashboard (like cronJobs does)
or
- Send the email below to the email of the recruiter:

                                           The Email Header logo
      =================================================================
Hi Achille,
Here is a candidate below interested in your Job profile below from JobApp You are receiving recommendations for **tester** in great **Levant,Maine** 
_Review and Update your preferences_ for a better match  

**_Contact this candidate_**

Candidate:
========
Achille,
1 year(s), tester
Skill Set : c,yy,cobol,fortran,pascal,c++,java
Country : United States
State : Massachusetts
City : great barrington

Your job post:
==========
Job Title : tester
Company Name : JobApp 
Country : United States
State : maine
City : levant

**_Contact this candidate_**

                                                        The email Footer goes here
===================================================================

**Comment by Astrea123** [comment]
It is confusing you have set title of the issue is "Contact this Recruiter" and  in the description you have written "Contact this Job Seeker"

**Comment by achille005** [comment]
That is exactly what i meant! You know we are limiting "Contact this Job Seeker" feature to only recruiters who have paid. correct? Now in "Contact this Recruiter" feature we are letting the Job seeker send their information to Recruiters for free - If i was a Recruiter i wont pay for "Contact this Job Seeker" feature, because i know Job seeker will contact me anyway...

In Summary
=========
When "Contact this Recruiter" button press, do 1 of the both solutions i propose above and change the "Contact this Recruiter" to " Recruiter Notified!" if the email is successfully sent.

**Comment by Astrea123** [comment]
We have already done this by changing text "Already Applied" on "Contact this Recruiter" button

**Comment by Astrea123** [comment]
![image](https://cloud.githubusercontent.com/assets/26182663/26052263/82593932-3982-11e7-8be0-6a7627b532ae.png)

**Comment by achille005** [comment]
In my summary above you address one piece of the issue. What about the content of the email being sent that i explained above. If it is not clear please let me know, we can call or i will explain again.

**Comment by Astrea123** [comment]
For above Section 
When JobSeeker will click on the "Contact this Recruiter" button then the content you have suggested will send to the Recruiter right ?
In that case there is no use of this window.
![image](https://cloud.githubusercontent.com/assets/26182663/26090660/10cad684-3a24-11e7-9962-825aa6cf0b83.png)

Then what should I do for that

**Comment by achille005** [comment]
Yes. Don't provide a window for the JS to type in stuffs. We just want to send the JobSeeker info and the job that match him to the R dashboard.
 (Make sure to include those fields, "Skills need for this job" and "Last updated date:"

**Comment by userfor544** [comment]
To elaborate: the idea is to send something like this table below to recruiter email ( when "Contact this Recruiter" is clicked). We already know what happened when "Contact this recruiter is clicked"

![image](https://cloud.githubusercontent.com/assets/6147947/26117686/50f83344-3a35-11e7-8626-ee570f91d2a3.png)

**Comment by achille005** [comment]
Can you put "Your Job Post" and " New Matching Candidate" in an html table like:

|Your Job  Alert Setup                          |                                                New Matching Candidate |
==============                                                                              ================  



=======================================================
Also you know
"Update your preference" is a link 
"JobApp" is a link
and 
"Contact this candidate" is a link and when clicked it should take the Recruiter to his dashboard normally with that matching candidate selected. 

If you can do all that then send the mail to the R's dashboard
NOTE: 
- jobapp.com is NOT our damin - our is jobapp.co 
for subject use( upper case, ! at the end and :: right after JOBAPP
- JOBAPP :: NEW CANDIDATE ALERT!

**Comment by Astrea123** [comment]
Can you send me an image that will be more clear about the html table ?
Also Please Specify how to show in html table if no matching alert found

**Comment by userfor544** [comment]
![image](https://cloud.githubusercontent.com/assets/6147947/26315740/198235e0-3ee0-11e7-8680-2381ed4374e8.png)

**Comment by Astrea123** [comment]
On "Contact this candidate" I have provided the link of Recruiter Login .We can not pass the recruiter's sensitive info in the mail . because the mail will send from Job Seeker Dashboard

**Comment by achille005** [comment]
That is a good idea to force the recruiter to login to contact the candidate but if the recruiter has many JS on his profile, he won't know is that JS. If after login you can show the matching JS profiles screen with the one he his contacting SELECTED or highlighted. that will be good.

**Comment by achille005** [comment]
I have fowarded you how the cron job email content are looking. Not visible. Also forwarded you few good none from indeed and capitalone. Here is simplier version:
- Make the orange button looks like a button 
- Job Seeker email content 
![new js email content](https://user-images.githubusercontent.com/6147947/27513068-02a68dec-5927-11e7-9185-89515e68da42.png)

- Recruiter
![new recruiter email content](https://user-images.githubusercontent.com/6147947/27513178-1f96c27a-592a-11e7-9ee6-d452190e11bc.png)

**Comment by achille005** [comment]
Waiting on a new cronjob email to verify this and close it.

**Comment by achille005** [comment]
- Also center the header image logo like the footer -  the position is a little off compared to the footer
- Also center the Content ( see how i did it above )
- keep the Thank you as it is.

Current format - it is Off
================== 
![image](https://user-images.githubusercontent.com/6147947/28067018-62a0c292-660d-11e7-8a15-335b81d5c6e5.png)
